scripts/cert_generation.sh

```diff
@@ -197,13 +214,28 @@ function generate_certs() {
 
 function generate_extensions() {
   local add_oid=$1
+  local add_localhost=$2
+  shift
   shift
   local cert_names=$@
 
-  extension_names="IP.1:127.0.0.1,DNS.1:localhost"
-  extension_index=2
+  extension_names=""
+  extension_index=1
+  local use_comma=0
+
+  if [ "$add_localhost" == "true" ]; then
+    extension_names="IP.1:127.0.0.1,DNS.1:localhost"
+    extension_index=2
+    use_comma=1
+  fi
+
   for name in ${cert_names//,/}; do
-    extension_names="${extension_names},DNS.${extension_index}:${name}"
+    if [ $use_comma = 1 ]; then
+      extension_names="${extension_names},DNS.${extension_index}:${name}"
+    else
+      extension_names="${extension_names}DNS.${extension_index}:${name}"
```

Comment:
ok - but since `$use_comma` is `0` here, you could just say `extension_names="DNS.${extension_index}:${name}"`